.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     executor:
       name: hmpps/localstack
       jdk_tag: "17.0"
-      localstack_tag: "2.3"
+      localstack_tag: "3"
```

Comment:
Needed to get the tests working otherwise all create queue requests to Localstack failed with a 500.